### PR TITLE
Add a small description and a minimal example to the Readme.md

### PR DESCRIPTION
**Pull request by Phenitei**
Nothing that can't be derived from reading the code here or the documentation of Blaze, but it's more beginner friendly imho like that -- I'd never used blaze and was a bit weirded out by the lack of any description at all.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
 # servant-lucid - Blaze-html support for servant
 
 ![servant](https://raw.githubusercontent.com/haskell-servant/servant/master/servant.png)
+
+This package allows you to use [blaze-html](http://hackage.haskell.org/package/blaze-html-0.8.0.0/docs/Text-Blaze-Html5.html) to serve html pages in your servant APIs. More specifically, it exports a `HTML` datatype with the correct `MimeRender` instances so that you can write `type myAPI = Get '[HTML] a`.
```
Comment (alpmestan) [inline README.md]: The name of the `type` must start with an uppercase and there's no `a` in scope, so I suggest to make things a bit clearer to write something like `type MyAPI = Get '[HTML] User` or something along those lines?

Comment (Phenitei) [reply README.md]: Right for the `User` instead of `a`; as for the typename... my bad, I thought I'd typed API for the type. Will fix it of course.

**Comment by Phenitei** [comment]
Is this better?

**Comment by alpmestan** [comment]
Thanks!

**Pull request merged**